java/gandiva/src/test/java/org/apache/arrow/gandiva/evaluator/ProjectorTest.java

```diff
@@ -1193,7 +1193,7 @@ public void testInExpr() throws GandivaException, Exception {
     Field c1 = Field.nullable("c1", int32);
 
     TreeNode inExpr =
-        TreeBuilder.makeInExpressionInt32(TreeBuilder.makeField(c1), Sets.newHashSet(1, 2, 3, 4, 5, 15, 16));
+            TreeBuilder.makeInExpressionInt32(TreeBuilder.makeField(c1), Sets.newHashSet(1, 2, 3, 4, 5, 15, 16));
```

Comment:
Is this indent style change intentional?
It seems that we use 4 spaces not 8 spaces for this case in other places.

Reply:
It was unintentional I'm sorry, my IDE was with a bad configuration, already fixed it.